### PR TITLE
[MRG] error if channel name too long

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mne/channels/channels.py [hunk 1]
```diff
@@ -871,6 +871,8 @@ def interpolate_bads(self, reset_bads=True, mode='accurate',
 def rename_channels(info, mapping):
     """Rename channels.
 
+    .. warning::  The channel names must have less than 15 characters
```
Comment (larsoner) [inline mne/channels/channels.py]: at most 15

**Comment by agramfort** [comment]
pushed

for the check give it a try. Make we is over ...

**Comment by larsoner** [comment]
Will do tomorrow

**Comment by larsoner** [comment]
Okay @agramfort changes pushed, see what you think

**Comment by agramfort** [comment]
thx @larsoner

**Comment by larsoner** [comment]
Probably should have waited for CIs to come back, but hopefully it passes :)

**Comment by larsoner** [comment]
Looks like they did not (failure is due to this PR though it shows up on a different one):

https://travis-ci.org/mne-tools/mne-python/jobs/279534080#L2437

I'll roll a fix into #4602 and merge once the CIs come back happy there.

**Pull request merged**